tests/functional/sca/test_scan_ci.py

```diff
@@ -0,0 +1,83 @@
+from pathlib import Path
+
+from tests.functional.utils import run_ggshield_sca_scan
+from tests.repository import Repository
+
+
+def test_scan_ci_diff(tmp_path: Path, monkeypatch, pipfile_lock_with_vuln) -> None:
+
```

Comment:
The usage is to have a docstring at the beginning of the test with the following format:
```
"""
GIVEN ...
WHEN ...
THEN ...
"""
# code here
```
so that we know directly what is tested.